modules/color-picker/react/lib/ColorInput.tsx

```diff
@@ -26,7 +26,6 @@ const swatchTileSpacing = spacing.xxs;
 const swatchTileSize = 20;
 const swatchCheckIconSpacing = 8;
 const colorInputWidth = 116;
-const colorInputErrorWidth = 144;
 
 const CustomHexInput = styled(TextInput)<Pick<ColorInputProps, 'disabled' | 'error' | 'grow'>>(
```

Comment:
Because `error` is no longer needed, could you remove this mention of `'error'` here as well as the `ErrorType` import on line 7 and in the interface on line 17?